jscomp/others/js_dict.ml

```diff
@@ -27,6 +27,7 @@ type 'a t
 type key = string
 
 external get : 'a t -> key -> 'a option = "" [@@bs.get_index] [@@bs.return {undefined_to_opt}]
+external unsafe_get : 'a t -> key -> 'a = "" [@@bs.get_index] 
```

Comment:
This API is consistent (both name and behavior) with OCaml other `unsafe` function such as `Array.unsafe_get`. Those functions are provided for performance reason when the caller has already checked for the conditions. 

In my use case the caller code, first calls `keys` function and then iterates over those keys. Therefore the check is already done and it is indeed safe to call `unsafe_get`. 

If we want a function which throws exception, then usually the convention in the OCaml community would be to have `exn_get` function. This one will do boundary checks and throws invalid_args if condition is not met.